src/locales/en.json

```diff
@@ -130,8 +131,13 @@
     "field_arrival_port": "Arrival port",
     "field_arrival_coordinates": "Arrival coordinates",
 
-    "field_port_hint": "Provide the IATA or ICAO code for the port. If the port does not have a code, enter '%s'. The arrival port must be different to the departure port.",
-    "field_coordinate_hint": "You are only required to enter coordinates if your port does not have an IATA or ICAO code. Provide coordinates in decimal format to 4 decimal places.",
+    "field_port_choice": "Do you know the departure port's ICAO or IATA code?",
+    "field_choice_yes": "Yes",
+    "field_choice_no": "No",
+    "field_port_code": "IATA or ICAO code",
+    "field_port_choice_hint": "If the International Civil Aviation Organisation (ICAO) or International Air Transport Association (IATA) code is unknown, geographic coordinates are expected.",
```

Comment:
An example of the explain abbreviation on first use element of the site. Whether this is to be adopted, or whether it will become a separate screen of terms, business still need to discuss.